Lib/ufo2ft/featureWriter/kernFeatureWriter.py

```diff
@@ -27,23 +25,14 @@ class KernFeatureWriter(object):
 
     leftFeaClassRe = r"@MMK_L_(.+)"
     rightFeaClassRe = r"@MMK_R_(.+)"
+    _font = None
 
-    def __init__(self, font):
-        self.font = font
+    def __init__(self, font=None, features=("kern", )):
```

Comment:
why is `font=None` an optional keyword argument and not a required positional argument like in the base class?
What does a `None` value means in this context?
And the `self.font` property setter, is that for reusing the same FeatureWriter instance with multiple fonts, or is it just a way to customize initialization in the FeatureWriter's subclasses? I would keep it simple and just require a not-None font instance to be passed in the constructor of any FeatureWriter, and instantiate one feature writer per font.

Reply:
I removed that bit.